lib/sidekiq/limit_fetch/queues.rb

```diff
@@ -35,8 +39,15 @@ def dynamic?
 
   def add(queues)
     queues.each do |queue|
-      @queues.push queue unless @queues.include? queue
+      unless @queues.include? queue
+        apply_process_limit_to_queue(queue)
+        apply_limit_to_queue(queue)
+
+        @queues.push queue
+      end
     end
+
+    apply_blocks_to_queues
```

Comment:
Thank you for the work you've done!
I have only one thing that concerns me in this PR.

We don't need to re-apply blocks from scratch here. Especially since `@blocks` are the initial blocks (from config) and some queues could have been blocked/unblocked afterwards therefore diverging from the state of `@blocks`.

Since this method `add` is used only to wire up new-found dynamic queues and there is no way initial blocks (from config) could have impact on it, I think, this line should be moved to the `Queues.start`. Dynamic queues will still have the ability to be blocked/unblocked on the fly. 

What do you think?